plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveDistributedQueries.java

```diff
@@ -58,6 +59,28 @@ public void testDelete()
         // Hive connector supports row-by-row delete only for ACID tables
     }
 
+    @Override
+    public void testDeleteEntireTable()
+    {
+        String tableName = "test_delete_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM orders", 15000);
+        assertUpdate("DELETE FROM " + tableName);
+        assertQuery("SELECT count(*) FROM " + tableName, "SELECT 0");
+        assertUpdate("DROP TABLE " + tableName);
+    }
+
+    @Override
+    public void testDeleteWithBigintEquality()
+    {
+        // Hive connector supports row-by-row delete only for ACID tables or when entire partition is deleted
```

Comment:
Can this test set up an ACID table?
i guess file metastore is not supported for ACID, right? cc @djsstarburst

Reply:
Sadly, that is correct @findepi